include/dali/imgcodec/image_codec.h

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Should filename be changed to `image_decoder.h`?

Reply:
Done